<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>accountManageDialog</class>
 <widget class="QDialog" name="accountManageDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#accountManageDialog{
border-image:url(:/icon/icon/timg.jpg);
}
#accountManageDialog *{
border-image:url();
}</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>150</y>
     <width>51</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: white;</string>
   </property>
   <property name="text">
    <string>卡号：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="passwordLineEdit">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>180</y>
     <width>190</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 135, 0);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="bindConfirmButton">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>220</y>
     <width>190</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: white;
background-color: rgb(255, 135, 0);
border-style: outset;
padding: 6px;
border-radius: 10px;</string>
   </property>
   <property name="text">
    <string>确认绑定</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>180</y>
     <width>40</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: white;</string>
   </property>
   <property name="text">
    <string>密码</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="accountNumLineEdit">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>150</y>
     <width>190</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 135, 0);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="returnButton">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>220</y>
     <width>80</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: white;
background-color: rgb(255, 135, 0);
border-style: outset;
padding: 6px;
border-radius: 10px;</string>
   </property>
   <property name="text">
    <string>取消/返回</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>60</y>
     <width>351</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>华文琥珀</family>
     <pointsize>30</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: white;</string>
   </property>
   <property name="text">
    <string>Bind With Account</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
